pkg/commands/drift/command.go

```diff
@@ -117,6 +119,10 @@ func (c *DetectIamDriftCommand) Flags() *cli.FlagSet {
 }
 
 func (c *DetectIamDriftCommand) Run(ctx context.Context, args []string) error {
+	mClient := metrics.FromContext(ctx)
+	cleanup := metricswrap.WriteMetric(ctx, mClient, "command_iam_detect_drift", 1)
+	defer cleanup()
```

Comment:
```suggestion
	defer metricswrap.WriteMetric(ctx, mClient, "command_iam_detect_drift", 1)()
```

Is there a reason that won't work?

Reply:
That would work fine. I guess I thought it was easier to reason about what is happening by aliasing the returned function rather than calling it directly.